data-model/src/main/scala/za/co/absa/enceladus/model/MappingTable.scala

```diff
@@ -76,4 +78,19 @@ case class MappingTable(name: String,
           AuditFieldName("schemaVersion", "Schema Version"))) ++
         super.getSeqFieldsAudit(newRecord, AuditFieldName("defaultMappingValue", "Default Mapping Value")))
   }
+
+  override def exportItem(): String = {
+    val defaultMappingValueJsonList: ArrayNode = objectMapperBase.valueToTree(defaultMappingValue.toArray)
+
+    val objectItemMapper = objectMapperRoot.`with`("item")
+
+    objectItemMapper.put("name", name)
+    description.map(d => objectItemMapper.put("description", d))
+    objectItemMapper.put("hdfsPath", hdfsPath)
+    objectItemMapper.put("schemaName", schemaName)
+    objectItemMapper.put("schemaVersion", schemaVersion)
```

Comment:
Comment on this got burried, I'm afraid. What's the point of the version, when there probably won't be any relevancy between the versions of export system and import system.
Although I might see some point. Would like to discuss. 😉

Reply:
- Double-checking if what you exported has what you need. 
- Calling an export on dataset XYZ version 3 and then getting the schema that you need by extracting the name and version from the JSON
- Default value